commandapi-platforms/commandapi-paper/commandapi-paper-plugin/src/main/resources/config.yml

```diff
@@ -0,0 +1,83 @@
+################################################################################
```

Comment:
Is this file necessary anymore? The config should be automatically generated now.